.github/workflows/docs.yml

```
@@ -0,0 +1,36 @@
+      - run: npm run docs:build
+        working-directory: website
+      - uses: actions/upload-pages-artifact@v3
+        with:
+          path: website/src/.vuepress/dist
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: build
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
```

actionlint (1.7.12)

[error] property "deployment" is not defined in object type {}

(expression)